app/models/track.rb

```diff
@@ -92,6 +92,10 @@ def average_test_duration
     git.average_test_duration + INFRASTRUCTURE_DURATION_S
   end
 
+  def new?
```

Comment:
Do you think it makes sense to add this to the `Track` model, or should this just reside in `SerializeTrack`?

Reply:
I think do it in SerializeTrack. If we ever then use it somewhere else, we can move it here, but I think keeping the models small is good.